sqlglot/dialects/presto.py

```diff
@@ -626,3 +626,27 @@ def create_sql(self, expression: exp.Create) -> str:
             if kind == "VIEW" and schema.expressions:
                 expression.this.set("expressions", None)
             return super().create_sql(expression)
+
+        def delete_sql(self, expression: exp.Delete) -> str:
+            """
+            Presto only support DELETE FROM a table, without alias, so we need to remove the unnecessary parts.
+            If the original DELETE statement contains more than one table to be deleted, just pick the first one.
```

Comment:
I'm not sure about picking the first one. We should definitely use `self.unsupported(...)`, in any case. Perhaps returning `""` is the safer choice here to avoid ignoring the remaining `DELETE`s.

cc @tobymao

Reply:
if there are multiple deletes, we should just not do anything and have presto fail

Reply:
so
 
if tables:
   return super()...

Reply:
if there are multiple deletes -> DELETE without the FROM keyword will be parsed into tables, but sometime the statement delete only a table, eg in bigquery: `DELETE project.dataset.table where 1=1`

So, there are multiple deletes happen when
```python
if tables and len(tables) > 1:
    return super().delete_sql()
```

Reply:
@georgesittas @tobymao I have updated the PR, thanks for the suggestion.